doc/source/whatsnew/v0.17.0.txt

```diff
@@ -835,6 +843,38 @@ Other API Changes
   raise ``ValueError``                All other public methods (names not beginning with underscores)
   ===============================     ===============================================================
 
+
+- The following exceptions in the gbq module will raise ``ValueError`` instead of ``IOError``:
+    ``AccessDenied``
+    ``InvalidColumnOrder``
+    ``InvalidPageToken``
+    ``InvalidQuery``
+    ``TermsOfServiceNotAccepted``
+    ``UnknownGBQException``
+
+- The following exceptions in the gbq module have been renamed/added:
```

Comment:
this separate list is not necessary, remove it. Only the 'custom' ones matter. You can put a list of these exceptions in a pointer in the doc, but don't list them.